CONTRIBUTING.md

```diff
@@ -49,7 +49,8 @@ Current Reviewers:
   - [@aquinoit](https://github.com/aquinoit)
   - [@Grubba27](https://github.com/Grubba27)
 - [@filipenevola](https://github.com/filipenevola)
-- [@StorytellerCZ](https://github.com/StorytellerCZ)
+- [@Grubba27](https://github.com/Grubba27)
```

Comment:
Duplicate here, Grubba27 is already added at the end of the list.

Reply:
Just fixed, tks!